index.html

```diff
@@ -556,6 +560,8 @@
           </li>
           <li>Let <var>request</var> be a new <a>PaymentRequest</a>.
           </li>
+          <li>Set <var>request</var>.<a>[[\paymentRequestID]]</a> to <var>paymentRequestID</var>.
```

Comment:
Yeah, it might be easier/preferable to just put this in PaymentDetails. Any objection to that? Seems like less work. :)